src/coreclr/src/vm/pefile.cpp

```diff
@@ -1921,8 +1922,25 @@ PEAssembly::PEAssembly(
     m_debugName.Normalize();
     m_pDebugName = m_debugName;
 #endif
-}
 
+#ifndef DACCESS_COMPILE
```

Comment:
The whole function is `!DACCESS_COMPILE`. This ifdef should not be needed.

Reply:
Strange, I have thought I have removed it already.